api/Document.json

```diff
@@ -168,10 +168,10 @@
               "version_added": "4"
             },
             "opera": {
-              "version_added": true
+              "version_added": "3"
```

Comment:
I did, yes -- after testing this one out, I then figured it made more sense to just create the ranged version rather than test every Opera edition.  I do plan on going back through and adding exact versions for Opera in the future!

Reply:
Alright, well if this one is verified then there's no reason to not keep it. The data will look inconsistent, but let's fix that with more real versions later :D